tools/interactive/interactivetool_panoply.xml

```diff
@@ -9,17 +9,17 @@
         </entry_point>
     </entry_points>
     <command><![CDATA[
-        current_pwd=`pwd` &&
-        ln -s $current_pwd /config &&
-	mkdir /config/home  &&
+        mkdir output &&
+        mkdir /config/home  &&
 	mkdir /config/home/output &&
 	export HOME=/config/home &&
-	ln -s '$infile' '/config/home/$infile.display_name' &&
+	cp '$infile' '/config/home/$infile.display_name' &&
 	mkdir /config/home/.gissjava &&
 	cd /config/home/.gissjava    &&
 	tar xvf /opt/PanoplyJ/colorbars.tar &&
-	cd /config/home &&
-	/init
+	cd - &&
+	/init &&
+	cp /config/home/output/*.png output/.
```

Comment:
What happens if there are no pngs? I guess this should be allowed to fail? You can use `| true` for example.

Reply:
Good point! Thank you.